test/bin/stress_testing.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+#
+# This script will enable stress scenarios to run MicroShift tests
+# with constricted resources
+
+set -uo pipefail
+
+SCRIPT_NAME=$(basename "$0")
+IP=$(nslookup "$(hostname)" | grep -v "#" | awk '/Address:/{print $2}')
+INTERFACE=$(ip route | grep default | awk -v ip="${IP}" '$0~ip{print $5}')
+
+WONDERSHAPER="/usr/local/bin/wondershaper"
+SPEEDTEST_CLI="/usr/local/bin/speedtest-cli"
```

Comment:
Where are these coming from? We need to add support for those in fetch_tools.sh rather than hard-coding their paths.

Reply:
These are the local paths to 2 scripts that will be downloaded if they don't exist when they are going to run. No installation is needed, only download them, so I don't think there's need to add it to `fetch_tools.sh` 

I'm fine to add them into `fetch_tools.sh`, but I think it would be better if this `stress_testing.sh` script does not depend on any other one.